src/azul/transformer.py

```diff
@@ -648,7 +648,7 @@ def accumulate(self, value) -> bool:
         """
         :return: True, if the given value increased the count of unique values
         """
-        return self.items.accumulate(value)
+        return self.set_accum.accumulate(value)
```

Comment:
This change belongs in the previous commit. If you'd rather not move the change, consider squashing these together.

Reply:
By our current, ill-defined standards, the history is a bit fragmented. All of the commits are tightly coupled, so much so that I don't foresee us reverting one of them and leaving the others. That's my goto criterion for deciding what goes into a commit: would I ever want to revert a change separately from another change. If yes, separate commit. Otherwise, same commit.